crowds-gossip-frontend/src/components/register/Register.jsx

```diff
@@ -0,0 +1,154 @@
+//bstakhdmo mkan el array 3alshan brag3 mno hagat is async byhsal 3aliha life cycle w athakm fiha
```

Comment:
butting franko in code not best practice you know but good work writing this code